forms/htmleditor/TinyMCEConfig.php

```diff
@@ -34,6 +34,7 @@ class TinyMCEConfig extends HtmlEditorConfig {
 		'remove_script_host' => true,
 		'convert_urls' => false, // Prevent site-root images being rewritten to base relative
 		'menubar' => false,
+		'language' => 'en',
```

Comment:
Ok, that's a good idea... as much "tinymce specific" content should really be inside this class, and any interface for "set current editor language" should probably be abstracted to the config abstract interface.